pkg/apis/kops/validation/aws.go

```diff
@@ -79,3 +83,23 @@ func awsValidateMachineType(fieldPath *field.Path, machineType string) field.Err
 
 	return allErrs
 }
+
+// TODO: make image validation smarter? graduate from jessie to stretch? This is quick and dirty because we keep getting reports
+func awsValidateAMIforNVMe(fieldPath *field.Path, ig *kops.InstanceGroup) field.ErrorList {
+	NVMe_INSTANCE_PREFIXES := []string{"P3", "C5", "M5", "H1"}
```

Comment:
I believe i3’s are also NVME

https://aws.amazon.com/ec2/instance-types/i3/

Reply:
I didn't mean to push this to kubernetes org :/ will fix this point and reopen. Any ideas for a better place to list these types of instances? this is already hard to maintain. I am not aware of any other lists of instances that have this type of metadata hanging around. Would adding something like that to the machinetypes list be better? 

I hope this is a validation that we do not need to do for too much longer.

Reply:
@geojaz Agreed. We certainly could have a NVME flag in machinetypes, just like we do for gpu? That said, hopefully we don't need it much longer, it may not be worth implementing there

Reply:
yeah, I punted on getting this into 1.10, and the next day I had two colleagues come to me with this exact problem. people are definitely running into it,